tsl/test/sql/continuous_aggs.sql

```diff
@@ -82,6 +82,9 @@ SET ROLE :ROLE_DEFAULT_PERM_USER;
 -- TEST2 ---
 DROP MATERIALIZED VIEW mat_m1;
 
+SHOW enable_partitionwise_aggregate;
```

Comment:
This is not the best test case for this change as the mat view does not have an aggregate with partials in internal form.  avg with numeric or bigint type will have partials in internal form. So could you  add a test case specifically for this: ( I haven't been able to locate one yet)? It is similar to the test case at line 408 but the avg should be on a numeric type instead. (The INSERT stmt uses the partialize_agg stmt).

Reply:
I see that sum(numeric) also has internal aggtranstype.

Reply:
The fix doesn't affect aggregate functions. Another approach, which I initial implemented, is to enable paritionwise aggregate in the PostgreSQL config. However, I don't think it will be appropriate. My recommendation is to do proper testing with comparison between disabled and enabled paritionwise in a separate PR.

Reply:
Yes, this would be ideal.